### PR TITLE
Switching generic cemeteries to SVG

### PR DESCRIPTION
**Pull request by kocio-pl**
Related to https://github.com/gravitystorm/openstreetmap-carto/issues/2045.

Warsaw, z19
Before
swbltrbh
After
fistvh5l

### CODE REVIEW

**Comment by pnorman** [comment]
The pattern size needs to be some integer division of 256.

btw, I did a cemetery pattern at https://github.com/kartotherian/brighmed/blob/master/symbols/patterns/cemetery.svg

File: symbols/grave_yard_generic.svg [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   id="svg4"
+   viewBox="0 0 18 18"
+   height="18"
+   width="18"
```
Comment (pnorman) [inline symbols/grave_yard_generic.svg]: To tile correctly, the tile size needs to be an integer multiple of the pattern size, so this needs to be a power of two

**Comment by kocio-pl** [comment]
Thanks! Brighmed pattern does not directly fit here:
hnkmczff
But reworked 16px version should be OK:
t0p3zzkx

BTW: Interesting enough, [current PNG version](https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/grave_yard_generic.png) is 16x16.

**Comment by kocio-pl** [comment]
Not needed due to https://github.com/gravitystorm/openstreetmap-carto/pull/2714 work.

**Pull request closed without merging**